src/System.Collections/tests/Performance/Perf.Dictionary.cs

```diff
@@ -20,8 +20,7 @@ public class Perf_Dictionary
             while (dict.Count < size)
             {
                 int key = rand.Next(500000, int.MaxValue);
-                if (!dict.ContainsKey(key))
-                    dict.Add(key, 0);
+                dict.TryAdd(key, 0);
```

Comment:
This one is a bit unfortunate. We shouldn't have made this change in the first place (https://github.com/dotnet/corefx/pull/17201), because this is a perf test... I would say we should revert this everywhere (including master and release/2.0.0), but that is probably a non-starter this late in the game. This is okay as-is, I suppose.